static/js/admin.js

```diff
@@ -323,6 +323,8 @@ function _setup_page() {
         realm_default_language: page_params.realm_default_language,
         realm_waiting_period_threshold: page_params.realm_waiting_period_threshold,
         is_admin: page_params.is_admin,
+        realm_icon_source: page_params.realm_icon_source,
+        realm_icon: page_params.realm_icon,
```

Comment:
`page_params.realm_icon` should be called `page_params.realm_icon_url`  to match the avatar naming scheme (and for clarity about what type it is).